integration/spark-common/src/main/scala/org/apache/carbondata/spark/load/DataLoadProcessBuilderOnSpark.scala

```diff
@@ -160,4 +162,112 @@ object DataLoadProcessBuilderOnSpark {
       Array((uniqueLoadStatusId, (loadMetadataDetails, executionErrors)))
     }
   }
+
+  /**
+   * use FileScanRDD to read input csv files
```

Comment:
change comment to mention this function creates a RDD that does reading of multiple CSV files

Reply:
fixed